projects/fmcomms11/Readme.md

```diff
@@ -0,0 +1,9 @@
+# FMCOMMS11 HDL Project
+
+Here are some pointers to help you:
+  * [Board Product Page](https://www.analog.com/ad-fmcomms11-ebz)
+  * Parts : [12-Bit, 2.6 GSPS/2.5 GSPS/2.0 GSPS, 1.3 V/2.5 V Analog-to-Digital Converter](https://www.analog.com/ad9625)
+  * Parts : [16-Bit, 12 GSPS, RF Digital-to-Analog Converters](https://www.analog.com/ad9162)
+  * Project Doc: https://wiki.analog.com/resources/eval/user-guides/ad-fmcomms11-ebz
+  * HDL Doc: https://wiki.analog.com/resources/eval/user-guides/ad-fmcomms11-ebz/reference_hdl
+  * Linux Drivers:
```

Comment:
If we have generic linux drivers for DAQ2/DAQ3 (DDS/ADC IP core), we should probably have them for this project also. Speaking in general probably we should alss point to the specific ADC/DAC driver or make it more generic and point to the linux page with all the drivers, if there is not specific linux page for the project.